src/qt/test/paymentservertests.cpp

```diff
@@ -74,31 +74,31 @@ void PaymentServerTests::paymentServerTests()
     std::vector<unsigned char> data = DecodeBase64(paymentrequest1_BASE64);
     SendCoinsRecipient r = handleRequest(server, data);
     QString merchant;
-    r.paymentRequest.getMerchant(caStore, merchant);
+    QVERIFY(r.paymentRequest.getMerchant(caStore, merchant) & PR_AUTHENTICATED);
     QCOMPARE(merchant, QString("testmerchant.org"));
 
     // Version of the above, with an expired certificate:
     data = DecodeBase64(paymentrequest2_BASE64);
     r = handleRequest(server, data);
-    r.paymentRequest.getMerchant(caStore, merchant);
+    QVERIFY(r.paymentRequest.getMerchant(caStore, merchant) | PR_ERROR);
     QCOMPARE(merchant, QString(""));
 
     // Long certificate chain:
     data = DecodeBase64(paymentrequest3_BASE64);
     r = handleRequest(server, data);
-    r.paymentRequest.getMerchant(caStore, merchant);
+    QVERIFY(r.paymentRequest.getMerchant(caStore, merchant) & PR_AUTHENTICATED);
     QCOMPARE(merchant, QString("testmerchant8.org"));
 
     // Long certificate chain, with an expired certificate in the middle:
     data = DecodeBase64(paymentrequest4_BASE64);
     r = handleRequest(server, data);
-    r.paymentRequest.getMerchant(caStore, merchant);
+    QVERIFY(r.paymentRequest.getMerchant(caStore, merchant) | PR_ERROR);
```

Comment:
QVERIFY(x) where x is not zero will pass.
x = result | PR_PERROR is always non-zero. An OR operations with a non-zero value always results in a non-zero value.